ocp_resources/serving_runtime.py

```diff
@@ -13,91 +13,149 @@ class ServingRuntime(NamespacedResource):
         disabled: Optional[bool] = None,
-        protocol_versions: Optional[List[str]] = None,
-        image_pull_secrets: Optional[List[Dict[str, Any]]] = None,
-        grpc_endpoint: Optional[int] = None,
-        grpc_data_endpoint: Optional[int] = None,
-        http_data_endpoint: Optional[int] = None,
+        grpc_data_endpoint: Optional[str] = "",
+        grpc_endpoint: Optional[str] = "",
+        http_data_endpoint: Optional[str] = "",
+        image_pull_secrets: Optional[List[Any]] = None,
+        spec_labels: Optional[Dict[str, Any]] = None,
+        multi_model: Optional[bool] = None,
+        node_selector: Optional[Dict[str, Any]] = None,
+        protocol_versions: Optional[List[Any]] = None,
         replicas: Optional[int] = None,
         storage_helper: Optional[Dict[str, Any]] = None,
-        built_in_adapter: Optional[Dict[str, Any]] = None,
+        supported_model_formats: Optional[List[Any]] = None,
+        tolerations: Optional[List[Any]] = None,
+        volumes: Optional[List[Any]] = None,
         **kwargs: Any,
     ) -> None:
         """
-        ServingRuntime object
-
         Args:
             containers (Optional[List[Dict[str, Any]]], mandatory if yaml file is not provided): Containers of the serving runtime.
+
             supported_model_formats (Optional[List[Dict[str, Any]]]): Model formats supported by the serving runtime.
+
             multi_model (Optional[bool]): Specifies if the model server can serve multiple models.
+
             disabled (Optional[bool]):  When set to true it disables the serving runtime.
+
             protocol_versions (Optional[List[str]]): List of protocols versions used by the serving runtime.
+
             image_pull_secrets (Optional[List[Dict[str, Any]]]): List of references to secrets used for pulling images used by this SR.
+
             grpc_endpoint (Optional[str]): Port of the gRPC endpoint.
+
             grpc_data_endpoint (Optional[str]): Port of the gRPC data endpoint.
+
             http_data_endpoint (Optional[str]): HTTP endpoint for inferencing.
+
             replicas (Optional[int]): Number of replicas of the deploymnent generated by this SR.
+
             storage_helper (Optional[Dict[str, Any]]): Configuration for this runtime's use of the storage helper.
+
             built_in_adapter Optional[Dict[str, Any]]: Configuration for the built-in adapter.
+
+            affinity (Dict[str, Any]): No field description from API; please add description
+
+            spec_annotations (Dict[str, Any]): No field description from API; please add description
+
+            spec_labels (Dict[str, Any]): No field description from API; please add description
+
+            node_selector (Dict[str, Any]): No field description from API; please add description
+
+            tolerations (List[Any]): No field description from API; please add description
+
+            volumes (List[Any]): No field description from API; please add description
```

Comment:
@tarukumar

Please help with the missing descriptions.

Reply:
please find the below desc:

`affinity (Optional[Dict[str, Any]]): Affinity is a group of affinity scheduling rules for model pod using SR.
 spec_annotations (Optional[Dict[str, str]]): Runtime Spec Resource annotations.
 spec_labels (Optional[Dict[str, str]]): Runtime Spec Resource label.
 node_selector (Optional[Dict[str, Any]]): Node selectors on  SR and labels on nodes to control where the model pod is scheduled.
 tolerations (Optional[List[Any]]): allow the scheduler to schedule model pods with matching taints in SR
 volumes(Optional[List[Any]]): List of volumes that can be mounted by containers belonging to the
              pod.`

Reply:
Thanks